packages/fetch-error-handler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@dotcom-reliability-kit/fetch-error-handler",
+  "version": "0.0.0",
```

Comment:
suggestion: would the version need to change to v0.1.0 as it's experimental? or is this set to v0.0.0 due to release-please?

Reply:
The version will get bumped to `v0.1.0` by Release Please once this gets merged into `main` 🙂 setting it to `0.1.0` in this PR would cause Release Please to tag up `0.2.0`.